src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac-image"
+        class="card__image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="card__product-code">
+        Product code:
+        <span id="card__product-id">195434</span>
+      </div>
+      <div class="card__reviews">
+        <ul class="stars card__reviews-rate">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <div class="card__reviews-amount">
+          Reviews:
+          <span id="reviews-amount">5</span>
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-title">Price:</div>
+        <div class="card__price-value">
+          $
+          <span id="card__price-value">2,199</span>
+        </div>
+      </div>
+      <a
+        href="#"
+        class="button card__button"
```

Comment:
why use two different classes here? make sure you checked checklist before submitting PR

you should have only 2 BEM blocks here